src/main/java/uk/gov/pay/webhooks/webhook/WebhookService.java

```diff
@@ -157,19 +158,32 @@ private boolean webhookHasSubscriptionForEvent(WebhookEntity webhook, InternalEv
                 .anyMatch(internalEventNames -> internalEventNames.contains(event.eventType()));
     }
 
-    public void deleteWebhookMessagesOlderThan() {
+    public void expireWebhookMessages() {
```

Comment:
I think `delete` in the name is fine. `expire` could be confused with status being expired